adoc/chapters/programming_interface.adoc

```diff
@@ -17218,15 +17218,15 @@ a@
 template <access::address_space AddressSpace, access::decorated IsDecorated>
 void load(size_t offset, multi_ptr<const DataT, AddressSpace, IsDecorated> ptr)
 ----
-   a@ Loads the values at the address of [code]#ptr# offset in elements of type [code]#DataT# by [code]#NumElements * offset#, into the components of this SYCL [code]#vec#.
+   a@ Loads the values at the address of [code]#ptr# offset in elements of type [code]#DataT# by [code]#NumElements * offset#, into the components of this SYCL [code]#vec#. [code]#vec# must be aligned to the size of the data type of this SYCL vec.
```

Comment:
Should be something like:

> The `ptr` must be aligned to `alignof(DataT)`.

Reply:
@gmlueck  sorry for the delay
Just pushed the necessary change. Let me know if any other changes would be required

Reply:
Thanks, that looks good.